packages/client/src/util/mrk.js

```diff
@@ -82,7 +82,6 @@ const withState = state => {
       if (look() === '#') {
         read()
 
-        let c
```

Comment:
Where is `c` defined? The `let c` from the previous block won't exist here - that `c` only exists in the scope of the block after `if (look() === '+')`. (`let` has no hoisting :) )

Reply:
For some reason uglifyjs complained that `c` was already defined when I did `yarn run build`.

Reply:
Yup, I did that.